cloudigrade/account/models.py

```diff
@@ -85,6 +107,18 @@ def __repr__(self):
         return f'<AwsInstance {self.ec2_instance_id}>'
 
 
+class AwsMachineImage(MachineImage):
+    """MachineImage model for an AWS EC2 instance."""
+
+    ec2_ami_id = models.CharField(
+        max_length=256,
```

Comment:
AMI IDs are 17 characters, with a 4 character prefix ("ami-"), for a total of 21 characters. (e.g. "i-08165b429be87c52e")

max_length can be reduced to 32 or 64 without any issue.

Reply:
We can definitely reduce. There's a pre-existing `AwsInstance` model with the same field, and I was matching length and field type here.

Reply:
👍 to @adberglund's comment. We have some wonky oversized fields at present, and eventually we should go tighten them up, but I think this is fine right now for sake of consistency. The big field cleanup can come later.